pkg/network/dhcp/configurator.go

```diff
@@ -9,15 +9,15 @@ import (
 	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
 )
 
-const dhcpStartedDirectory = "/var/run/kubevirt-private"
+const defaultDHCPStartedDirectory = "/var/run/kubevirt-private"
 
 type Configurator struct {
 	advertisingIfaceName string
 	cacheFactory         cache.InterfaceCacheFactory
 	filterByMac          bool
 	handler              netdriver.NetworkHandler
 	launcherPID          string
-	dhcpStartedDirectory string
+	DHCPStartedDirectory string
```

Comment:
I think this is a result of search/replace here, right? I'd rather name the field `dhcpStartedDirectory` again.

Reply:
No. We need to expose this attribute so we can inject the directory where the files will be created in the unit tests.

It sucks, but it's the simplest way. And it could suck more.